### PR TITLE
Enable alternative display form selection feature flag for Tiger

### PR DESCRIPTION
**Pull request by xMort**
JIRA: TNT-628

---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/8140/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tests/218/ : SUCCESS in 10m 30s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/8139/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/9961/ : SUCCESS in 17m 25s

File: libs/sdk-ui-filters/src/AttributeFilter/utils/AttributeFilterUtils.ts [hunk 1]
```diff
@@ -137,6 +137,7 @@ export const getElementTotalCount = async (
         .forDisplayForm(objRef)
         .withOptions({
             ...(searchString ? { filter: searchString } : {}),
+            excludePrimaryLabel: true,
```
Comment (kandl) [inline libs/sdk-ui-filters/src/AttributeFilter/utils/AttributeFilterUtils.ts:140]: Is this correct for bear? (or shoud we use some backend capability for this?)

Comment (xMort) [reply libs/sdk-ui-filters/src/AttributeFilter/utils/AttributeFilterUtils.ts:140]: Bear does not implement this option. It is applied only on Tiger, Bear ignores it. The fact is explained in the TSDoc. Is it OK like this? I'm not sure how to handle it differently.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Review by kandl — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR2565

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/2326/ : SUCCESS in 1m 18s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/2619/ : SUCCESS in 32s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/1438/ : SUCCESS in 18m 40s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/9975/ : SUCCESS in 17m 08s

**Pull request merged**